src/components/media/controls/controls.tsx

```diff
@@ -9,23 +9,41 @@ import IconButton from '@mui/material/IconButton'
 import LinearProgress from '@mui/material/LinearProgress'
 import Slider from '@mui/material/Slider'
 import Typography from '@mui/material/Typography'
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 
 import { formatDurationTime } from '../../helper'
 
 interface MediaIconButtonProps {
   onClick: () => void
-  action: 'play' | 'pause' | 'forward' | 'replay' | 'volumeUp' | 'volumeOff'
+  action:
+    | 'play'
+    | 'pause'
+    | 'forward'
+    | 'replay'
+    | 'volumeUp'
+    | 'volumeOff'
+    | 'pictureInPicture'
+    | 'pictureInPictureExit'
+    | 'fullscreen'
+    | 'fullscreenExit'
   className?: string
+  color?: string
```

Comment:
why do we need to pass the color as prop? can we not set it to 'primary' always ?

Reply:
The colours change based on the component as well as the viewport, so I was trying to think of a way to control this.

Reply:
No, the colors should not change by viewport. Please keep this to primary